### PR TITLE
PRO-2685: Add google translate widget

### PR DESCRIPTION
**Pull request by sehgxl**
What does the pr do?
- Adds google translate widget.

Concerns : 
- This [guide](https://docs.google.com/document/d/12umeKgbOzmLEgm2PitGM-MBAEtycJVhNFkoreXjb4DY/edit) was shared by the the client. According to this the API key gets exposed. 

Screenshots
image

### CODE REVIEW

**Review by mikevic — COMMENTED** [review]
@sehgxl - we should talk to Karan or the client on where they want the widget to be placed?

File: src/index.html [hunk 1]
```diff
@@ -91,6 +91,20 @@
...
+    function googleTranslateElementInit() {
+      new google.translate.TranslateElement(
+        {
+          pageLanguage: "en",
+          layout: google.translate.TranslateElement.InlineLayout.SIMPLE,
+        },
+        "google_translate_element"
+      );
+    }
+    console.log("google script ran");
```
Comment (gpalsingh) [inline src/index.html:104]: Remove console logs

**Review by gpalsingh — CHANGES_REQUESTED** [review]

**Review by gpalsingh — APPROVED** [review]

**Review by gpalsingh — CHANGES_REQUESTED** [review]
Since the issue was related to CSP, can we move back the code to `index.html`?

**Comment by sehgxl** [comment]
> Since the issue was related to CSP, can we move back the code to `index.html`?

I tried doing that. It does not work when deployed. So again I am adding the tag dynamically.

File: src/index.html [hunk 2]
```diff
@@ -89,8 +89,7 @@
 
   gtag('config', 'UA-114435812-1');
   </script> -->
-  <!-- Including Ck-edior script-->
-  <script src="https://cdn.ckeditor.com/4.6.1/full/ckeditor.js"></script>
```
Comment (gpalsingh) [inline src/index.html:93]: Why is this being removed?

Comment (gpalsingh) [reply src/index.html:93]: @sehgxl

**Pull request closed without merging**